docs_src/microservices/V2TopLevelMigration.md

```diff
@@ -0,0 +1,97 @@
+# V2 Migration Guide
+
+!!! edgey "EdgeX 2.0"
+    Many backward breaking changes occurred in the EdgeX 2.0 (Ireland) release which may require some migration depending on your use case.
+
+This section describes how to migrate from EdgeX 1.x to EdgeX 2.0 at a high level and refers the reader to the appropriate detail documents. The areas to consider for migrating are:
+
+- [Compose File](#compose-file)
+- [Database](#database)
+- [Custom Configuration](#custom-configuration)
+- [Custom Device Service](#custom-device-service)
+- [Custom Applications Service](#custom-applications-service)
+
+## Custom Compose File
+
+The compose files for V2 have many changes from their V1 counter parts. If you have customized a V1 compose file to add additional services or tweak overrides,  it is highly recommended that you start with the appropriate V2 compose file and re-add your customizations. It is very likely that the sections for your additional services will need to be migrated to have the proper environment overrides. Best approach is to use one of the V2 service sections that closest matches your service (App or Device) as a template.
+
+## Database
+
+There currently is no migration path for the data stored in the database. The V2 data collections are stored separately from the V1 data collections in the Redis database. Redis is now the only supported database, i.e. support for Mongo has been removed.
+
+!!! note
+    Since the V1 data and V2 data are stored separately, one could create a migration tool and upstream it to the EdgeX community.
+
+!!! warning
+    If the database is not cleared before starting the V2 services, the old V1 data will still reside in the database taking up useful memory. It is recommended that you first wipe the database clean before starting EdgeX 2.0 Services. That is unless you create a DB migration tool.
```

Comment:
If we are going to make this statement, then I think we should provide the ways they could do this such as destroy the volume, provide the V1 REST APIs, or provide Redis CLI commands to wipe it, etc.

Reply:
Agreed for volume and V1 Rest APIs. 
Using Redis CLI/Client will only work if running non-secure since the Redis password is randomly generated in secure mode.

Reply:
Looked at the REST APIs and that is tedious way. So for now just described the volume removal approach.